pysat/instruments/jro_isr.py

```diff
@@ -93,7 +94,7 @@
 # To ensure this function is always applied first, we set the filter
 # function as the default function for (JRO).
 # Default function is run first by the nanokernel on every load call.
-default = pysat.instruments.madrigal_methods.filter_data_single_date      
+default = mad_meth.filter_data_single_date      
```

Comment:
ha, made me laugh.

Reply:
Me too.  Can't take credit for that one though.

Reply:
Glad I could brighten up some days :)